### PR TITLE
Fix Scrubber

### PR DESCRIPTION
**Pull request by engelhartrueben**
# Fixes # (issue)
Scrubbing numbers with Twilio

## Description
Upgraded Twilio Lookup to V2. [Twilio Documentation](https://www.twilio.com/docs/lookup/v2-api)

```terminal
// From V1
phoneNumber:  {
  callingCountryCode: '1',
  countryCode: 'US',
  phoneNumber: '+~~~~~~~~~~~',
  nationalFormat: '~~~~~~~~~~~',
  valid: true,
  validationErrors: [],
  callerName: null,
  simSwap: null,
  callForwarding: null,
  lineStatus: null,
  lineTypeIntelligence: null,
  identityMatch: null,
  reassignedNumber: null,
  smsPumpingRisk: null,
  phoneNumberQualityScore: null,
  url: 'https://lookups.twilio.com/v2/PhoneNumbers/+~~~~~~~~~'
}

// To V2
phoneNumber:  {
  callingCountryCode: '1',
  countryCode: 'US',
  phoneNumber: '~~~~~~~~~~',
  nationalFormat: '~~~~~~~~~~',
  valid: true,
  validationErrors: [],
  callerName: null,
  simSwap: null,
  callForwarding: null,
  lineStatus: null,
  lineTypeIntelligence: {
    carrier_name: 'Verizon Wireless',
    error_code: null,
    mobile_country_code: '310',
    mobile_network_code: '010',
    type: 'mobile'
  },
  identityMatch: null,
  reassignedNumber: null,
  smsPumpingRisk: null,
  phoneNumberQualityScore: null,
  url: 'https://lookups.twilio.com/v2/PhoneNumbers/~~~~~~~~~~~'
}
```

_Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change, and any blockers that make your change a WIP_

# Checklist:

- [x] I have manually tested my changes on desktop and mobile (and staging)
- [x] The test suite passes locally with my changes
- [ ] If my change is a UI change, I have attached a screenshot to the description section of this pull request
- [x] [My change is 300 lines of code or less](https://github.com/StateVoicesNational/Spoke/blob/main/CONTRIBUTING.md#submitting-your-pull-request), or has a documented reason in the description why it’s longer
- [ ] I have made any necessary changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] My PR is labeled [WIP] if it is in progress

### CODE REVIEW

File: src/extensions/service-vendors/twilio/index.js [hunk 1]
```diff
@@ -633,44 +633,46 @@ export async function getContactInfo({
     return {};
   }
   const twilio = await exports.getTwilio(organization);
-  const types = ["carrier"];
+  const types = { fields: "line_type_intelligence" };
   if (lookupName) {
     // caller-name is more expensive
-    types.push("caller-name");
+    types.fields = ("line_type_intelligence,caller-name");
```
Comment (mau11) [inline src/extensions/service-vendors/twilio/index.js]: should this be an array instead of parentheses?

Comment (engelhartrueben) [reply src/extensions/service-vendors/twilio/index.js]: Oddly enough, no: https://www.twilio.com/docs/lookup/v2-api#code-lookup-with-data-packages
Twilio wants `types.fields` as a comma separated string. That parentheses is awkward though

File: src/extensions/service-managers/scrub-bad-mobilenums/index.js [hunk 2]
```diff
@@ -306,6 +306,7 @@ export async function nextBatchJobLookups({
       lastCount,
       steps
     );
+    console.log("scrub-bad-mobilenums error: ", err);
```
Comment (engelhartrueben) [inline src/extensions/service-managers/scrub-bad-mobilenums/index.js]: log.error

**Comment by engelhartrueben** [comment]
@mau11 the changes we talked about are in.

**Review by mau11 — APPROVED** [review]
🎉

**Pull request merged**